tardis/tests/tests_slow/conftest.py

```diff
@@ -33,13 +42,23 @@ def pytest_configure(config):
 
 
 def pytest_unconfigure(config):
-    integration_tests_configpath = config.getvalue("integration-tests")
-    if integration_tests_configpath is not None:
+    # Unregister only if it was registered in pytest_configure
+    if config.option.dokureport_active:
         config.pluginmanager.unregister(config.dokureport)
```

Comment:
Usually third party plugins have this `pytest_unconfigure`, where there are unregistered, once their job is done. As they do not assume anything outside in the main test run. But in our case, I think we can remove this part.